doc/market-gr/MyData-fiskaltrust-Mappings.md

```diff
@@ -0,0 +1,208 @@
+# myDATA to fiskaltrust Type Mappings
+
+This document outlines the mappings between myDATA (Greek tax authority AADE) specific datatypes and fiskaltrust types based on the actual implementation in `AADEMappings.cs`.
+
+## fiskaltrust Hexcode Structure
+
+The fiskaltrust hexcodes follow a specific 64-bit structure for Greek localization:
+
+```
+0x4752_2000_0000_0000
+  └┴┴┤ └┴┴┤ └┴┴┤ └┴┴┴ BaseCase
+     │    │    ╰───── Flags/Nature/Service Type
+     │    ╰────────── Version (v2 = 0x2000)
+     ╰─────────────── Country Code (GR = 0x4752)
+```
+
+### Country Code
+- `0x4752` = Greece (GR in ASCII hex)
+
+### Version
+- `0x2000` = Version 2 of the interface
+
+### Flags and Modifiers
+- **Receipt Flags**: `0x0010_0000` = Refund flag
+- **Service Types**: 
+  - `0x0100_0000` = Other Service
+  - `0x0200_0000` = Not Own Sales  
+  - `0x0300_0000` = Receivable
```

Comment:
This is very Incorrect or I don't understand it ^^

These are for one chargeitem cases which is not mentioned.
And also 
```cs
OtherService = 0x20,
Receivable = 0x90,
NotOwnSales = 0x60,
```

also the position of the type of service flag is `0x0000_0000_0000_00F0`.